pkg/manager/manager.go

```diff
@@ -378,3 +379,57 @@ func (m *SupportBundleManager) getNodeSelector() map[string]string {
 	}
 	return nodeSelector
 }
+
+func (m *SupportBundleManager) getTaintToleration() []v1.Toleration {
+	taintToleration := []v1.Toleration{}
+
+	m.TaintToleration = strings.ReplaceAll(m.TaintToleration, " ", "")
+	if m.TaintToleration == "" {
+		return taintToleration
+	}
+
+	tolerationList := strings.Split(m.TaintToleration, ",")
+	for _, toleration := range tolerationList {
+		toleration, err := parseToleration(toleration)
+		if err != nil {
+			logrus.WithError(err).Warnf("Invalid toleration: %s", toleration)
+			continue
+		}
+		taintToleration = append(taintToleration, *toleration)
+	}
+	return taintToleration
+}
+
+func parseToleration(taintToleration string) (*v1.Toleration, error) {
```

Comment:
I can't find a good library for this. Could we have some unit tests if possible?

Reply:
Done!